specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json

```diff
@@ -1179,7 +1179,7 @@
           "default": {
             "description": "Error response describing why the operation failed.",
             "schema": {
-              "$ref": "#/definitions/ErrorResponse"
+              "$ref": "#/definitions/HighCasedErrorResponse"
```

Comment:
How about other references to `ErrorResponse`? Do they need to be updated either?

Reply:
@raych1 our APIs currently return both kind of Error Response - upper case one and the camelcasing ones. So keeping both these.